cmake/Buildcodyco-modules.cmake

```diff
@@ -26,4 +26,8 @@ ycm_ep_helper(codyco-modules TYPE GIT
               COMPONENT dynamics
               FOLDER robotology
               DEPENDS ${codyco-module_DEPENDS}
-              CMAKE_ARGS -DCODYCO_USES_KDL:BOOL=OFF)
+              CMAKE_ARGS -DCODYCO_USES_KDL:BOOL=OFF
+                         -DENABLE_codycomod_genericSensorClient:BOOL=OFF
```

Comment:
I see that here you disable the codyco devices. What remains from codyco that we still use ? Or is it an intermediate step before removing codyco?

Reply:
> 
> 
> I see that here you disable the codyco devices. What remains from codyco that we still use ? Or is it an intermediate step before removing codyco?

The open points are the yarpmotorgui files with the positions of the robot, see https://github.com/robotology/codyco-modules/issues/273 .

Reply:
I see. I followed that discussion and thought it was solved I guess still has to be released. I just pinged on that.